pyriemann/utils/tangentspace.py

```diff
@@ -396,6 +482,7 @@ def unupper(T):
     "logchol": log_map_logchol,
     "logeuclid": log_map_logeuclid,
     "riemann": log_map_riemann,
+    "wasserstein": log_map_wasserstein,
```

Comment:
Can you add `"wasserstein"` in the description of `metric` for `tangent_space` and `untangent_space`?